Makefile

```diff
@@ -54,6 +54,10 @@ clean: ## Cleanup
 	@rm -rf .tox
 
 auto-style: ## Style the code
+	@if type isort >/dev/null 2>&1 ; then isort -rc . ; \
```

Comment:
@AlexanderWillner Just played around with `isort`. Works great. Adding it to my personal workflow.

Found this flag was needed:
` --lines-after-imports 2`

Also, I'd propose to specify the more verbose version of the flag (e.g. `--check-only`). That makes it clearer when skimming over the code. Also I can't see the `-r` flag anywhere.